ppr-ui/src/store/state/state-model.ts

```diff
@@ -250,5 +262,19 @@ export const stateModel: StateModelIF = {
       validateSteps: false,
       validateApp: false
     }
+  },
+  mhrTransferValidationState: {
+    homeOwnersValid: {
+      ownersValid: false
+    },
+    reviewConfirmValid: {
+      validateApp: false
+    }
+  },
+  mhrTransfer: {
+    mhrNumber: '',
+    ownerGroups: [],
+    owners: [],
```

Comment:
Do we need `owners` here?

Reply:
Yea, I thought it was ported over by default, but after cleanup, we don't need the owners (no use). So we can remove it from here as well. Thanks!

Reply:
Sounds good, thanks! Although i will remove in the next ticket. There is several State updates to happen with respect to the Transfer Schema, so it makes sense 👍

Reply:
Sounds good, resolved!